### PR TITLE
Remove jwpsf module, PEP8 cleanup

### PR DESCRIPTION
**Pull request by jdavies-st**
- Remove `jwpsf` module that has been tagging along in this package even though development stopped 9 years ago on it.  It is fully replaced by [`webbpsf`](https://github.com/spacetelescope/webbpsf).  That's ~1500 lines of code that were not tested now gone from the package.  Sorry @philhodge!  😃 
- Turn on `flake8` checking in `setup.cfg` and fix PEP8 issues (and real bugs) for:
   - `jwst/lib`
   - `ramp_fitting`
   - `exp_to_source`

### CODE REVIEW

**Comment by jdavies-st** [comment]
@dmggh any idea what this import is?

https://github.com/spacetelescope/jwst/blob/55e757d1f7f2ca3b04b955a0af43649a773fa61f/jwst/ramp_fitting/mc_2d.py#L46

Is this module used at all?  How about the 3d version?

**Comment by dmggh** [comment]
This module (and the 3D version) are not used - I will remove them.


________________________________

File: jwst/exp_to_source/tests/test_exp_to_source.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 import numpy as np
 
 from . import helpers
-from ...datamodels import (MultiExposureModel, MultiSlitModel, ModelContainer)
-from ..exp_to_source import exp_to_source, multislit_to_container
+from jwst.datamodels import (MultiExposureModel, MultiSlitModel, ModelContainer)
```
Comment (hbushouse) [inline jwst/exp_to_source/tests/test_exp_to_source.py]: I thought the rule for this package was to due relative imports, rather than absolute.

Comment (jdavies-st) [reply jwst/exp_to_source/tests/test_exp_to_source.py]: This is a test, and tests should have absolute imports.  That said, we have not been consistent here, so we have a bit of a mixed bag in our tests.

**Comment by dmggh** [comment]
I have removed the obsolete modules compare_cr_navg_files.py,  compare_crs.py,  create_cube.py,  mc_2d.py,  mc_3d.py,  rotate_dataset.py,  rotate_utils.py, and compare_cr_files.py from the repo.

**Comment by jdavies-st** [comment]
Addresses #2552

**Pull request merged**